AnnoyancesFilter/sections/mobile-app.txt

```diff
@@ -36,6 +36,7 @@ decrypt.co#?##__next > div[class^="sc-"] > div[class^="sc-"] > div[class^="sc-"]
 video.fc2.com##.c-app-sp2
 zhihu.com#$?#.ContentItem > div.ModalWrap:has(> div > div.ModalExp-content > div.ModalWrap-content > div:contains(继续浏览内容)) { display: none!important; }
 zhihu.com#$#body { overflow: visible!important; }
+pinterest.com##div[style*="transition: transform 225ms ease-in-out 0s; transform: translateY(0%);"]
```

Comment:
this rule doesn't work on my device

Reply:
![Screenshot 2021-03-15 at 15 15 45](https://user-images.githubusercontent.com/18536328/111152284-9a4e8c80-85a1-11eb-8ac7-ee2d4954fbb5.png)

Reply:
ok, now I see